nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/ConsumeJMS.java

```diff
@@ -54,8 +61,31 @@
 @SeeAlso(value = { PublishJMS.class, JMSConnectionFactoryProvider.class })
 public class ConsumeJMS extends AbstractJMSProcessor<JMSConsumer> {
 
+    static final AllowableValue AUTO_ACK = new AllowableValue(String.valueOf(Session.AUTO_ACKNOWLEDGE),
```

Comment:
@olegz would it be worthwhile to change the AllowableValue displayNames to "Auto Acknowledge (1)" and "Client Acknowledge (2)" (for example) or am I just overthinking this?